draft-ietf-quic-recovery.md

```diff
@@ -350,17 +350,20 @@ endpoint:
   min_rtt.  This limits the underestimation that a misreporting peer can cause
   to the smoothed_rtt.
 
-On the first RTT sample for a network path, the smoothed_rtt is set to the
-latest_rtt.
-
-smoothed_rtt and rttvar are computed as follows, similar to {{?RFC6298}}.  On
-the first RTT sample for a network path:
+smoothed_rtt and rttvar are computed as follows, similar to {{?RFC6298}}.  When
+there are no samples for a network path:
 
 ~~~
-smoothed_rtt = latest_rtt
-rttvar = latest_rtt / 2
+smoothed_rtt = rtt_sample
+rttvar = rtt_sample / 2
 ~~~
 
+where rtt_sample is the initial RTT.
+
+On the first RTT sample for the network path, smoothed_rtt and rttvar are set to
+the values using the formula above, using that first RTT sample as rtt_sample.
+This ensures a measurement erases the history of any persisted or default value.
```

Comment:
```suggestion
This ensures that the first measurement erases the history of any persisted or default value.
```

Reply:
Thank you for the suggestion. Applied in c4ed9aa.